anesthetic/plot.py

```diff
@@ -82,7 +88,8 @@ def make_1d_axes(params, **kwargs):
     return fig, axes
 
 
-def make_2d_axes(params, **kwargs):
+def make_2d_axes(params, types={'1d': 'kde', '2d': ['kde', 'scatter']},
+                 **kwargs):
```

Comment:
I'm not sure this change is an improvement, and does result in different behaviour from before: could you revert it for now (leaving the previous behaviour with a 'diagonal' keyword argument), and submit it as a separate pull request if you think that it's worth including.

Reply:
How should `diagonal` and `types` or `plot_type` interact? Should `diagonal` turn the `'1d'` keyword on and off?

Reply:
I agree that if ``'1d'`` is not present, then that should imply ``diagonal=False``, but I think that if a user were to provide just a string as a ``plot_type``, then by default both 1d and 2d plots should be produced, and ``diagonal=True`` by default